### PR TITLE
Add documentation to non-comparison helpers

### PR DESCRIPTION
**Pull request by tylersticka**
Closes issue #5.

For once, I have nothing more to say that the PR title doesn't already cover. :sweat_smile:

### CODE REVIEW

File: lib/random.js [hunk 1]
```diff
@@ -1,8 +1,20 @@
 'use strict';
 
+/**
+ * Return only one random item from an array.
+ *
+ * @since v0.0.1
+ * @param {Array} arr
+ * @return One random item.
```
Comment (erikjung) [inline lib/random.js]: Is there a proper way to denote _any_ data type?

File: lib/toFixed.js [hunk 2]
```diff
@@ -1,5 +1,16 @@
...
+ * Strip decimal points from a number.
+ *
+ * @since v0.0.1
+ * @param {Number} num
+ * @return {Number}
+ * @example
+ *
+ *   {{toFixed 19.99}} //=> 19
+ */
+
```
Comment (erikjung) [inline lib/toFixed.js]: I think this is backwards, actually. It should _add_ the decimals. So `1` passed in would yield `"1.00"`.

File: lib/random.js (hunk 1 above)
Comment (tylersticka) [inline lib/random.js]: @erikjung Not that I can find. Some articles say the type is optional, others say it isn't. I haven't found anything concrete yet.

**Comment by erikjung** [comment]
:+1: LGTM

**Pull request merged**